hydroshare/settings.py

```diff
@@ -708,6 +708,9 @@
     'hs_core.tasks.unzip_task': 'file unzip',
     'hs_core.tasks.copy_resource_task': 'resource copy',
 }
+
+HSWS_ACTIVATED = False
```

Comment:
Added this to settings.py because without this setting, some code will error trying to read it.